packages/ember-glimmer/lib/renderer.js

```diff
@@ -40,6 +40,63 @@ class DynamicScope {
   }
 }
 
+let nextRootId = 0;
+class RootState {
+  constructor(env, root, template, self, parentElement, dynamicScope) {
+    assert(`You cannot render \`${self.value()}\` without a template.`, template);
+
+    this.id = ++nextRootId;
```

Comment:
Ya, probably.  I am not really using this `id` at all, I was mostly just using it during debugging to make sure I was looking at the same `RootState` instance.